includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -94,8 +94,8 @@ public function read( &$coupon ) {
 		$coupon->set_props( array(
 			'code'                        => $post_object->post_title,
 			'description'                 => $post_object->post_excerpt,
-			'date_created'                => strtotime( $post_object->post_date_gmt ),
-			'date_modified'               => strtotime( $post_object->post_modified_gmt ),
+			'date_created'                => 0 < $post_object->post_date_gmt ? wc_string_to_timestamp( $post_object->post_date_gmt ) : null,
```

Comment:
This is to prevent it creating a timestamp from 0000.. so you don't get invalid times. It should be null. @claudiulodro is that wrong?